package/yast2-rdp.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Mar 18 14:05:05 UTC 2019 - knut.anderssen@suse.com
+
+- Remove network dependency completely
```

Comment:
bsc is missing, not sure what to refer, or maybe just wait until a real issue (or fill a new bug about the not needed dependency(